app/assets/stylesheets/argo.scss

```diff
@@ -0,0 +1,6 @@
+@import 'jquery-ui/custom-theme/jquery-ui-1.8.13.custom';
+@import './argo/*';
+ 
+.ui-jqgrid td { border: none }
+
+.argo-obj-status-highlight { color: #00F }
```

Comment:
Color literals like `#00F` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#00F` should be written as `#00f`<br>Declaration should be terminated by a semicolon